scripts/lib.sh

```diff
@@ -162,6 +162,8 @@ node-version-check() {
     node -pe "process.exit(Number(!(process.version == 'v' + require('./package.json').volta.node )))" ||
         (
             echo 'Unexpected node version. Recommended to use https://github.com/volta-cli/volta'
+            echo 'Run `volta install node` and `volta install yarn` to update your toolchain.'
```

Comment:
Are these steps needed? I thought volta reads the config and will auto install it for you.

Reply:
They shouldn't be required, but I thought this could be a good backup plan in-case a dev has busted their toolchains up.